src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -1115,11 +1115,14 @@ public KojiBuildInfo getBuildInfo( int buildId, KojiSessionInfo session )
     private KojiBuildInfo withGAV( KojiBuildInfo buildInfo, KojiSessionInfo session )
             throws KojiClientException
     {
+        if (true) return buildInfo;
```

Comment:
This line of code needs to be cleaned up, but it's a minor inefficiency right now. I'll leave it, and merge this.